app/src/main/java/com/cornellappdev/android/eatery/network/JsonUtilities.java

```diff
@@ -75,4 +79,41 @@ public static String loadJSONFromAsset(Context context, String fileName) {
 		}
 		return eateryList;
 	}
+
+    public static ArrayList<EateryBaseModel> parseEateries(List<AllEateriesQuery.Eatery> eateries, Context mainContext) {
+        ArrayList<EateryBaseModel> eateryList = new ArrayList<EateryBaseModel>();
+        JSONObject hardCodedData = null;
+        try {
+            hardCodedData = new JSONObject(loadJSONFromAsset(mainContext, "externalEateries.json"));
+            JSONArray hardCodedEateries = hardCodedData.getJSONArray("eateries");
+            for (int i = 0; i < hardCodedEateries.length(); i++) {
+                JSONObject obj = hardCodedEateries.getJSONObject(i);
+                EateryBaseModel model;
+                if (obj.has("id") && DINING_HALL_IDS.contains(obj.getInt("id"))) {
+                    model = DiningHallModel.fromJSON(mainContext, true, obj);
+                } else {
+                    model = CafeModel.fromJSONObject(mainContext, true, obj);
+                }
+                if (model != null) {
+                    eateryList.add(model);
+                }
+            }
+            for (int i = 0; i < eateries.size(); i++) {
```

Comment:
same question here, we're not using `i` in the for loop

Reply:
Changed to use an iterator.